app/src/main/java/com/example/giftly/handler/FireBaseClient.java

```diff
@@ -409,9 +405,13 @@ public ArrayList<Event> call() {
             try {
                 Tasks.await(callDB);
                 for (DocumentSnapshot event : callDB.getResult())
-                    retrievedUsers.add(new Event(event));
+                    retrievedUsers.add(EventBuilder.createEvent(event));
                 return retrievedUsers;
-            } catch (Exception e) {
+            } catch (ExecutionException e) {
+                Log.d(TAG, "ERROR MAKING EVENT LIST: " + e);
```

Comment:
Since the string is similar to the second catch statement, maybe specify the error clearly to avoid confusion.